src/main/java/com/nina/modelling/shoes/ShoeFactory.java

```diff
@@ -0,0 +1,39 @@
+package com.nina.modelling.shoes;
+
+
+public class ShoeFactory extends Shoe {
+
+    public static void main(String[] args) {
+
+        KidShoe kidShoe = new KidShoe("Summer", "Flower", "Orthopedic");
```

Comment:
can be didShoeOrthopedic

Reply:
What do you mean? The name of variable must be name didShoeOrthopedic?